trace_context/TRACE_STATE_REGISTRY.md

```diff
@@ -0,0 +1,7 @@
+# Trace State Registry
```

Comment:
I think something like this is two-fold: one is format and another is well-known-public-provider.

For example: aws might be referring to their format? It could also be referring to the public cloud itself.

It is worth considering. Also, I think the first format on the list should be equivalent to the `traceparent` maybe the label `tc` for native trace-context format.

Reply:
reason I use the amazon example is that early days "ec2" was both an api and also a public service. People started cloning the api. In the jclouds project, we had to come up with a way to describe the api separate from the provider of it. Incidentally we used the same registry for both. Ex "aws-ec2" was a provider of "ec2", but both labels were in the same registry for ease of use.

Reply:
@adriancole I just want to setup a registry for format, which relates to public service provider, vendor or project. And each on of them can choose a name and keep it public to else.

Reply:
@adriancole such as, if AWS support this sepc, their X-RAY need a tracestate name in value field. They should choose their own.

Reply:
so @wu-sheng you think to put entire header as described in the link as a trace state?

Reply:
@SergeyKanzhelev I think that could be optional, if this format is open. This maybe help people to diagnose interop issues.